LINUX/io.github.horsicq.detect-it-easy.metainfo.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 horsicq -->
+<component type="desktop-application">
+  <id>io.github.horsicq.detect-it-easy</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <name>Detect It Easy</name>
+  <developer_name>horsicq</developer_name>
+  <update_contact>horsicq@gmail.com</update_contact>
+  <summary>Advanced file analyzer</summary>
+  <description>
+    <p>Detect It Easy, or abbreviated "DIE" is a program to analyze files you load into the application.</p>
+    <p>It detects, among other things, the compiler and linker used, signatures, and other information about files.</p>
+    <p>The program has been designed specifically to reveal file signatures and how they are packed.</p>
+    <p>Some of the binary formats supported:</p>
+    <ul>
+      <li>MS-DOS: DOS MZ</li>
+      <li>Windows: Portable Executable (PE)</li>
+      <li>*Nix: Executable and Linkable Format (ELF)</li>
+      <li>macOS: Mach Object (Mach-O)</li>
+    </ul>
+  </description>
+  <launchable type="desktop-id">io.github.horsicq.detect-it-easy.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/guihkx/flatpak-app-detect-it-easy/ef62b241881138f43ce7004b5a6760d5f6c24deb/screenshots/main_window_basic_view.png</image>
```

Comment:
This screenshot, and the one below this one, are currently being hosted [on my repository](https://github.com/guihkx/flatpak-app-detect-it-easy/tree/master/screenshots), which is not ideal, but should be fine for now.

In any case, let me know if you have a better place to host them right now (or if you just want take different screenshots yourself).

Here's the [recommended guidelines](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#qsr-app-screenshots-info) for them.

Reply:
It is Ok! :)